pass-authz-acl/src/main/java/org/dataconservancy/pass/authz/acl/PolicyEngine.java

```diff
@@ -70,14 +72,12 @@ public void updateSubmission(URI uri) {
         final Set<URI> authReaders = new HashSet<>();
         final Set<URI> authWriters = new HashSet<>();
 
-        if (submission.getUser() != null) {
-            authReaders.add(submission.getUser());
+        // If a submission is "submitted=true", then it's frozen.
+        if (submission.getSubmitted() == null || !submission.getSubmitted()) {
 
-            // If a submission is "submitted=true", then it's frozen.
-            if (submission.getSubmitted() == null || !submission.getSubmitted()) {
-                // Not frozen, allow writes
-                authWriters.add(submission.getUser());
-            }
+            // Not frozen, allow writes to preparers and submitters
+            ofNullable(submission.getSubmitter()).ifPresent(authWriters::add);
+            submission.getPreparers().forEach(authWriters::add);
```

Comment:
This looks like it might not work correctly if `submission.getSubmitter()` is a `mailto:` URI?